### PR TITLE
Update boring-cyborg configuration

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

For a more precise definition of labels.

## Type of change

Please delete options that are not relevant.

- [x] CI system update

# How Has This Been Tested?

After applying this PR, the next PR should be parsed more correctly 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/boring-cyborg.yml [hunk 1]
```diff
@@ -25,6 +30,24 @@ labelPRBasedOnFilePath:
...
     - 'script/**/*'
 
+  # Add 'security' label to any change within the 'securemgr' folder or any subfolders and 'SECURITY.md' file
+  security:
+    - 'internal/controller/securemgr/**/*'
+    - '.github/SECURITY.md'
+
+  # Add 'Web UI' label to any change within the 'web' folder or any subfolders
+  Web UI:
+    - 'web/**/*'
```
Comment (t25kim) [inline .github/boring-cyborg.yml:40]: Please add `internal/webui` folder.

**Review by t25kim — APPROVED** [review]
LGTM Thanks!

File: .github/boring-cyborg.yml (hunk 1 above)
Comment (tdrozdovsky) [reply .github/boring-cyborg.yml:40]: Done!

**Comment by tdrozdovsky** [comment]
Yesterday Eric Ball installed the boring-cyborg for our repository

image

result:
image

**Review by MoonkiHong — APPROVED** [review]
+1

**Review by suresh-lc — APPROVED** [review]
LGTM

**Pull request merged**